cclunit-framework-source/src/main/resources/cclutmock_table.inc

```diff
@@ -0,0 +1,728 @@
+                                set cclutMockColumnIndex = cclutMockColumnIndex + 1
+                                if (cclutMockDataLength != cclutMockDataIndex + 4)
+                                    set cclutMockDataIndex = cclutMockDataIndex + 4
+                                endif
+                                set cclutNullColumn = TRUE
+                        else
+                            ;\null must be by itself
+                            call cclexception(100, "E",
+                                concat("cclutAddMockData() - \null column must have no other characters. rowData: ",
+                                    CCLUT_ROW_DATA))
+                            call parser(" reset ")
+                            call parser(" set modify system go ")
+                            return
+                        endif
+                    else
+                        ;Invalid escape character
+                        call cclexception(100, "E", concat("cclutAddMockData() - Invalid escape sequence. rowData: ",
+                            CCLUT_ROW_DATA))
+                        call parser(" reset ")
+                        call parser(" set modify system go ")
+                        return
+                    endif
+                endif
+            ;All other characters
+            else
+                set cclutDataBuffer = notrim(concat(cclutDataBuffer, cclutDataCharacter))
+            endif
+        endfor
+    endif
+
+    if (cclutMockColumnIndex <= cclutMockColumnLength)
+        call cclexception(100, "E", concat("cclutAddMockData() - Too few columns of data. rowData: ", CCLUT_ROW_DATA))
+        call parser(" reset ")
+        call parser(" set modify system go ")
+        return
+    endif
+
+    ;No columns have been added meaning all are null
+    if (cclutHasComma = 0)
+        call parser(" reset ")
+        call parser(concat(" rdb insert into ", cclut_mockTables->tables[cclutMockTablePos].replaceName, " values ("))
+        for (cclutMockColumnIndex = 1 to cclutMockColumnLength)
+            call parser(evaluate(cclutMockColumnIndex, 1, "NULL", ",NULL"))
+        endfor
+        call parser(") end ")
+    endif
+
+    call parser(" go ")
+end ;cclutAddMockData
+
+/**
+Helper function to parse out a column for a mock insertion.
+**/
+subroutine cclut::cclutParseColumn(tableIndex, columnIndex, dataBuffer)
+    declare CCLUT_COLUMN_COUNT = i4 with protect, constant(size(cclut_mockTables->tables[tableIndex].columns, 5))
+    declare cclutMockColumnType = vc with protect, noconstant("")
+    declare cclutMockColumnName = vc with protect, noconstant("")
+
+    if (columnIndex > CCLUT_COLUMN_COUNT)
+        call cclexception(100, "E", concat("cclutParseColumn() - Too many columns of data added to table. tableName: ",
+            cclut_mockTables->tables[tableIndex].originalName))
+        call parser(" reset ")
+        call parser(" set modify system go ")
```

Comment:
While working on this, I came across some odd irregularities with reset in parser and the use of it with respect to partial and full parser invocations that follow it.  

The quick explanation is that the next parser commands following this reset are some conditional parser statements invoked by the framework used within the assert() subroutines, followed by my full drop commands used within the teardown.

Using a full statement after reset corrects these problems.  I chose set modify system specifically because it was a no-op.  However, I've posted on the uCern group to get some answers to the internals of parser and why this works.  If it's not supported, then I'll modify the reset statements in some other way (maybe saving off the statements and doing a batch call only at the end).  Discussion is here: https://connect.ucern.com/message/6877662#6877662 (proprietary link again)